modules/logging-loki-retention.adoc

```diff
@@ -46,9 +44,47 @@ spec:
 ----
 <1> Sets retention policy for all log streams. *Note: This field does not impact the retention period for stored logs in object storage.*
 <2> Retention is enabled in the cluster when this block is added to the CR.
-<3> Contains the link:https://grafana.com/docs/loki/latest/logql/query_examples/#query-examples[LogQL query] used to define the log stream.
-+
-.Example per-tenant stream-based retention
+<3> Contains the link:https://grafana.com/docs/loki/latest/logql/query_examples/#query-examples[LogQL query] used to define the log stream.spec:
+  limits:
+ 
+.Example global stream-based retention for GCP
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+   global: <1>
```

Comment:
This should be under the `spec` property. `spec` is missing here
For ref. https://docs.openshift.com/container-platform/4.15/observability/logging/log_storage/cluster-logging-loki.html#logging-loki-retention_cluster-logging-loki

Reply:
Also @eohartman, quick question regarding the point 6 on your PR description, We already have a section on retention under official [docs](https://docs.openshift.com/dedicated/observability/logging/log_storage/cluster-logging-loki.html#logging-loki-retention_cluster-logging-loki) Any reason why we are adding it again for GCP? It seems redundant.

Reply:
Added the `spec` property and removed GCP examples.